jsapp/js/components/textFormatter/textFormatter.component.tsx

```diff
@@ -0,0 +1,56 @@
+import type {ReactNode} from 'react';
+
+const regExpMatchParts = /[^*[\]]+|(\*\*[^*]*(?:\*[^*]+)*\*\*)|(\*[^*]+\*)|(\[.+?\]\(.+?\)({:.+})?)/g;
+
+const processText = (text: string): ReactNode[] => {
+  const parts = text.match(regExpMatchParts);
+
+  if (!parts) {return [text];}
+
+  const formattedParts: ReactNode[] = [];
+
+  for (const part of parts) {
+    if (part.startsWith('**')) {
```

Comment:
Just a style issue, but we generally like to avoid if/else chains in favor of multiple simple if statements:

```
if (x) {
    return a
}
if (y) {
    return b
}
return c
```